client/src/components/PickupDetail.js

```diff
@@ -44,10 +50,10 @@ const PickupDetail: () => React$Node = () => {
         </div>
       </div>
       <div class="container-fluid m-1">
-        <a role="button" href="/add-pickup" className="btn btn-primary mx-1">
+        <a role="button" href="/" className="btn btn-primary mx-1">
```

Comment:
Why do these two buttons have different ways of redirecting? Use the same one for both.